server/views/pages/confirm-instructions.njk

```diff
@@ -175,6 +174,6 @@
 
 {% block javascripts %}
     <script src="/assets/accessible-autocomplete.min.js"></script>
-    <script src="/assets/js/autocomplete-email.js"></script>
+    <script id="autocomplete-script" src="/assets/js/autocomplete-email.js" {%- if scrollToBottom %} data-scroll-to-bottom="true" {% endif %}></script>
```

Comment:
it would be nice to have a test for this... if indeed we can verify that it has scrolled to the bottom